Clean the pyqgis nitpick list from conf.py file (#3687)
qgis/QGIS-Documentation: source/docs/pyqgis_developer_cookbook/crs.rst

  QGIS uses three different IDs for every reference system:

>   * :attr:`InternalCrsId <qgis.core.QgsCoordinateReferenceSystem.InternalCrsId>` --- IDs internally used in QGIS database.

Reviewer: ```suggestion
  * :attr:`InternalCrsId <qgis.core.QgsCoordinateReferenceSystem.InternalCrsId>` --- IDs used in the internal QGIS database.
```

>   * :attr:`PostgisCrsId <qgis.core.QgsCoordinateReferenceSystem.PostgisCrsId>` --- IDs used within PostGIS databases.

Reviewer: ```suggestion
  * :attr:`PostgisCrsId <qgis.core.QgsCoordinateReferenceSystem.PostgisCrsId>` --- ID used in PostGIS databases.
```

>   * :attr:`EpsgCrsId <qgis.core.QgsCoordinateReferenceSystem.EpsgCrsId>` --- IDs assigned by the EPSG organization

Reviewer: ```suggestion
  * :attr:`EpsgCrsId <qgis.core.QgsCoordinateReferenceSystem.EpsgCrsId>` --- ID assigned by the EPSG organization.
```

>   If not specified otherwise in second parameter, PostGIS SRID is used by default.

Reviewer: ```suggestion
  If not specified otherwise with the second parameter, PostGIS SRID is used by default.
```

     crs = QgsCoordinateReferenceSystem(wkt)
     assert crs.isValid()

> * create invalid CRS and then use one of the ``create*`` functions to

Reviewer: ```suggestion
* create an invalid CRS and then use one of the ``create*`` functions to
```

>   initialize it. In the following example we use Proj4 string to initialize the

Reviewer: ```suggestion
  initialize it. In the following example we use a Proj4 string to initialize the
```

>   projection

Reviewer: ```suggestion
  projection.
```

     assert crs.isValid()

It's wise to check whether creation (i.e. lookup in the database) of the CRS
has been successful: :meth:`isValid() <qgis.core.QgsCoordinateReferenceSystem.isValid>`
must return ``True``.

Note that for initialization of spatial reference systems QGIS needs to look up
appropriate values in its internal database :file:`srs.db`. Thus in case you
create an independent application you need to set paths correctly with

> :meth:`QgsApplication.setPrefixPath() <qgis.core.QgsApplication.setPrefixPath>`

Reviewer: ```suggestion
:meth:`QgsApplication.setPrefixPath() <qgis.core.QgsApplication.setPrefixPath>`,
```

otherwise it will fail to find the

> database. If you are running the commands from QGIS python console or

Reviewer: ```suggestion
database. If you are running the commands from the QGIS python console or
```

     assert crs.isValid()

It's wise to check whether creation (i.e. lookup in the database) of the CRS
has been successful: :meth:`isValid() <qgis.core.QgsCoordinateReferenceSystem.isValid>`
must return ``True``.

Note that for initialization of spatial reference systems QGIS needs to look up
appropriate values in its internal database :file:`srs.db`. Thus in case you
create an independent application you need to set paths correctly with
:meth:`QgsApplication.setPrefixPath() <qgis.core.QgsApplication.setPrefixPath>`,
otherwise it will fail to find the

> database. If you are running the commands from the QGIS python console or

Reviewer: ```suggestion
database. If you are running the commands from the QGIS Python console or
```
